packages/hw-app-ada/src/Ada.js

```diff
@@ -53,6 +53,37 @@ export type OutputTypeChange = {|
   path: BIP32Path
 |};
 
+export type Flags = {|
+ isDebug: boolean
+|};
+
+export type GetVersionResponse = {|
+  major: string,
+  minor: string,
+  patch: string,
+  flags: Flags
+|};
+
+export type DeriveAddressResponse = {|
+  address58: string
+|};
+
+export type GetExtendedPublicKeyResponse = {|
+  publicKeyHex: string,
+  chainCodeHex: string
+|};
+
+export type Witness = {|
+  path: BIP32Path,
+  witnessHex: string
```

Comment:
FYI: This is my problem I will have to fix in some subsequent PR -- instead fo `witnessHex` this should be `signatureHex` as the witness is not a full witness, just a signature